.github/workflows/ci.yaml

```diff
@@ -0,0 +1,51 @@
+name: kubectl plugin test
+on:
+  pull_request:
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+jobs:
+  Plugin-test:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: setup minikube
+      uses: manusa/actions-setup-minikube@v2.4.2
+      with:
+        minikube version: 'v1.21.0'
+        kubernetes version: 'v1.19.2'
+        start args: --memory 6g --cpus=2
+        github token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: print k8s cluster status
+      run:  |
+        minikube status
+        kubectl get nodes
+
+    - name: use local disk
+      run: tests/github-action-helper.sh use_local_disk
+
+    - name: deploy rook cluster
+      run: tests/github-action-helper.sh deploy_rook
+
+    - name: wait for operator pod to be ready
+      run: tests/github-action-helper.sh wait_for_pod_to_be_ready_state
+
+    - name: Test pluging with ceph commands
+      run: |
+        pwd
+        sudo mv kubectl-rook /usr/local/bin/kubectl-rook
```

Comment:
so `kubectl` looks for anything starting with kubectl in /usr/local/bin/? And that's it it works?

Reply:
> so `kubectl` looks for anything starting with kubectl in /usr/local/bin/? And that's it it works?

kubectl looks for `kubectl-` and anyname after `-` will be plugin name. in this case it is `kubectl-rook` so `rook` will be the plugin name.

there are a few limitations with `kubectl` plugin that's krew overcomes like naming the tool

Reply:
few docs to clear this:
1. https://krew.sigs.k8s.io/docs/developer-guide/develop/naming-guide/
2. https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/#naming-a-plugin
3. https://krew.sigs.k8s.io/docs/developer-guide/distributing-with-krew/